src/main/java/org/openstreetmap/atlas/utilities/vectortiles/TippecanoeCommands.java

```diff
@@ -0,0 +1,146 @@
+
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.utilities.runtime.RunScript;
+import org.openstreetmap.atlas.utilities.runtime.SingleLineMonitor;
+import org.openstreetmap.atlas.utilities.time.Time;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This utility class allows you to execute the tippecanoe command line tool ultimately via Java's
+ * runtime exec.
+ *
+ * @author hallahan
+ */
+public final class TippecanoeCommands
+{
+    private TippecanoeCommands()
+    {
+        // Utility class
+    }
+
+    private static final Logger logger = LoggerFactory.getLogger(TippecanoeCommands.class);
+    private static final SingleLineMonitor MONITOR = new SingleLineMonitor()
+    {
+        @Override
+        protected Optional<String> parseResult(final String line)
+        {
+            return Optional.of(line);
+        }
+    };
+
+    /**
+     * Allows you to check if you have installed tippecanoe on your system and if it satisfies the
+     * minimum version that we need.
+     *
+     * @return Whether you have a valid tippecanoe ready to execute in your environment.
+     */
+    public static boolean hasValidTippecanoe()
+    {
+        final String[] commandArray = new String[] { "tippecanoe", "--version" };
+        try
+        {
+            RunScript.run(commandArray, Collections.singletonList(MONITOR));
+        }
+        // When you look up the version, tippecanoe exits with 1, so getting here is normal.
+        // We want to get into this catch.
+        catch (final CoreException exception)
+        {
+            final Optional<String> result = MONITOR.getResult();
+            if (result.isPresent())
+            {
+                final String outputString = result.get();
+                final String[] versionArray = outputString.split("\n")[0].split("tippecanoe v");
+                // Here we extract the version.
+                if (versionArray.length == 2)
+                {
+                    final String versionString = versionArray[1];
+                    final DefaultArtifactVersion version = new DefaultArtifactVersion(
+                            versionString);
+                    if (TippecanoeSettings.MIN_VERSION.compareTo(version) <= 0)
+                    {
+                        return true;
+                    }
+                    else
+                    {
+                        logger.error(
+                                "Your version of tippecanoe is too old! The minimum version is {}",
+                                TippecanoeSettings.MIN_VERSION);
+                    }
+                }
+            }
+        }
```

Comment:
It might be worth adding some sort of "unexpected tippecanoe return value" or "failed to parse" log messages here and inside the `catch` block at the end.

That way if `tippecanoe` is ever patched to fix the `--version` exit code, or if the format of `--version` changes you won't just get a silent failure.